docker-compose.yml

```diff
@@ -1,19 +1,70 @@
 services:
-  mysql:
+  mysql_auth:
     # Always specify what version, "latest" causes issues later on
     image: mysql:9.1
-    container_name: dev_db
     environment:
-      MYSQL_USER: ${MYSQLDB_USER}
-      MYSQL_PASSWORD: ${MYSQLDB_PASSWORD}
-      MYSQL_ROOT_PASSWORD: ${MYSQLDB_PASSWORD_ROOT}
-      MYSQL_DATABASE: ${MYSQLDB_NAME}
+#      MYSQL_USER: ${AUTH_MYSQL_USERNAME}
+      MYSQL_PASSWORD: ${AUTH_MYSQL_PASSWORD}
+      MYSQL_ROOT_PASSWORD: ${AUTH_MYSQL_ROOT_PASSWORD}
+      MYSQL_DATABASE: ${AUTH_DB_NAME}
     networks:
       - net
     ports:
-      - "${MYSQLDB_PORT}:3306"
+      - "${AUTH_DOCKER_DB_PORT}:${AUTH_DOCKER_DB_PORT}"
```

Comment:
Dont fuck with the port on teh right.  
That is teh port that mysql runs on inside the docker image

https://github.com/Third-Floor-CSIS/cs4337-Big-Data-Group/blob/2e9640fbee6ef0e7e7b23cf47ecb9f40e8bbf250/docker-compose.yml#L13